GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day2/Formatting.java

```diff
@@ -0,0 +1,22 @@
+package hackkerank.day2;
+import java.util.Scanner;
+
+public class Formatting {
+
+    public static final String SPACE = "================================";
+    public static final String STYLE = "%-15s%03d%n";
+
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        System.out.println(SPACE);
+        for(int i=0;i<3;i++)
```

Comment:
Even for-loop variables must be descriptive and meaningful

Avoid magic numbers (i < 3) you can define a constant

Reply:
ok